keyboards/qneatandtidy/dx65/solder/rev1/info.json

```diff
@@ -0,0 +1,394 @@
+                { "label": "Slash", "matrix": [1, 13], "w": 1.5, "x": 13.5, "y": 1 },
+                { "label": "PgUp", "matrix": [1, 14], "x": 15, "y": 1 },
+
+                { "label": "Caps Lock", "matrix": [2, 0], "w": 1.75, "x": 0, "y": 2 },
+                { "label": "A", "matrix": [2, 1], "x": 1.75, "y": 2 },
+                { "label": "S", "matrix": [2, 2], "x": 2.75, "y": 2 },
+                { "label": "D", "matrix": [2, 3], "x": 3.75, "y": 2 },
+                { "label": "F", "matrix": [2, 4], "x": 4.75, "y": 2 },
+                { "label": "G", "matrix": [2, 5], "x": 5.75, "y": 2 },
+                { "label": "H", "matrix": [2, 6], "x": 6.75, "y": 2 },
+                { "label": "J", "matrix": [2, 7], "x": 7.75, "y": 2 },
+                { "label": "K", "matrix": [2, 8], "x": 8.75, "y": 2 },
+                { "label": "L", "matrix": [2, 9], "x": 9.75, "y": 2 },
+                { "label": "Scolon", "matrix": [2, 10], "x": 10.75, "y": 2 },
+                { "label": "Quote", "matrix": [2, 11], "x": 11.75, "y": 2 },
+                { "label": "Enter", "matrix": [2, 13], "w": 2.25, "x": 12.75, "y": 2 },
+                { "label": "PgDn", "matrix": [2, 14], "x": 15, "y": 2 },
+
+                { "label": "Shift", "matrix": [3, 0], "w": 2.25, "x": 0, "y": 3 },
+                { "label": "Z", "matrix": [3, 2], "x": 2.25, "y": 3 },
+                { "label": "X", "matrix": [3, 3], "x": 3.25, "y": 3 },
+                { "label": "C", "matrix": [3, 4], "x": 4.25, "y": 3 },
+                { "label": "V", "matrix": [3, 5], "x": 5.25, "y": 3 },
+                { "label": "B", "matrix": [3, 6], "x": 6.25, "y": 3 },
+                { "label": "N", "matrix": [3, 7], "x": 7.25, "y": 3 },
+                { "label": "M", "matrix": [3, 8], "x": 8.25, "y": 3 },
+                { "label": "Comma", "matrix": [3, 9], "x": 9.25, "y": 3 },
+                { "label": "Dot", "matrix": [3, 10], "x": 10.25, "y": 3 },
+                { "label": "Back Slash", "matrix": [3, 11], "x": 11.25, "y": 3 },
+                { "label": "Shift", "matrix": [3, 12], "w": 1.75, "x": 12.25, "y": 3 },
+                { "label": "Up", "matrix": [3, 13], "x": 14, "y": 3 },
+                { "label": "Fn", "matrix": [3, 14], "x": 15, "y": 3 },
+
+                { "label": "Ctrl", "matrix": [4, 0], "w": 1.25, "x": 0, "y": 4 },
+                { "label": "Gui", "matrix": [4, 1], "w": 1.25, "x": 1.25, "y": 4 },
+                { "label": "Alt", "matrix": [4, 2], "w": 1.25, "x": 2.5, "y": 4 },
+                { "label": "Space", "matrix": [4, 6], "w": 6.25, "x": 3.75, "y": 4 },
+                { "label": "Gui", "matrix": [4, 10], "x": 11, "y": 4 },
+                { "label": "Left", "matrix": [4, 12], "x": 13, "y": 4 },
+                { "label": "Down", "matrix": [4, 13], "x": 14, "y": 4 },
+                { "label": "Right", "matrix": [4, 14], "x": 15, "y": 4 }
+            ]
+        }
```

Comment:
```suggestion
        },
        "LAYOUT_65_ansi_blocker_tsangan_split_bs": {
            "layout": [
                { "label": "Esc", "matrix": [0, 0], "x": 0, "y": 0 },
                { "label": "1", "matrix": [0, 1], "x": 1, "y": 0 },
                { "label": "2", "matrix": [0, 2], "x": 2, "y": 0 },
                { "label": "3", "matrix": [0, 3], "x": 3, "y": 0 },
                { "label": "4", "matrix": [0, 4], "x": 4, "y": 0 },
                { "label": "5", "matrix": [0, 5], "x": 5, "y": 0 },
                { "label": "6", "matrix": [0, 6], "x": 6, "y": 0 },
                { "label": "7", "matrix": [0, 7], "x": 7, "y": 0 },
                { "label": "8", "matrix": [0, 8], "x": 8, "y": 0 },
                { "label": "9", "matrix": [0, 9], "x": 9, "y": 0 },
                { "label": "0", "matrix": [0, 10], "x": 10, "y": 0 },
                { "label": "Minus", "matrix": [0, 11], "x": 11, "y": 0 },
                { "label": "Equal", "matrix": [0, 12], "x": 12, "y": 0 },
                { "label": "Backspace", "matrix": [0, 13], "x": 13, "y": 0 },
                { "label": "Del", "matrix": [2, 12], "x": 14, "y": 0 },
                { "label": "PrtSc", "matrix": [0, 14], "x": 15, "y": 0 },

                { "label": "Tab", "matrix": [1, 0], "w": 1.5, "x": 0, "y": 1 },
                { "label": "Q", "matrix": [1, 1], "x": 1.5, "y": 1 },
                { "label": "W", "matrix": [1, 2], "x": 2.5, "y": 1 },
                { "label": "E", "matrix": [1, 3], "x": 3.5, "y": 1 },
                { "label": "R", "matrix": [1, 4], "x": 4.5, "y": 1 },
                { "label": "T", "matrix": [1, 5], "x": 5.5, "y": 1 },
                { "label": "Y", "matrix": [1, 6], "x": 6.5, "y": 1 },
                { "label": "U", "matrix": [1, 7], "x": 7.5, "y": 1 },
                { "label": "I", "matrix": [1, 8], "x": 8.5, "y": 1 },
                { "label": "O", "matrix": [1, 9], "x": 9.5, "y": 1 },
                { "label": "P", "matrix": [1, 10], "x": 10.5, "y": 1 },
                { "label": "Left Bracket", "matrix": [1, 11], "x": 11.5, "y": 1 },
                { "label": "Right Bracket", "matrix": [1, 12], "x": 12.5, "y": 1 },
                { "label": "Slash", "matrix": [1, 13], "w": 1.5, "x": 13.5, "y": 1 },
                { "label": "PgUp", "matrix": [1, 14], "x": 15, "y": 1 },

                { "label": "Caps Lock", "matrix": [2, 0], "w": 1.75, "x": 0, "y": 2 },
                { "label": "A", "matrix": [2, 1], "x": 1.75, "y": 2 },
                { "label": "S", "matrix": [2, 2], "x": 2.75, "y": 2 },
                { "label": "D", "matrix": [2, 3], "x": 3.75, "y": 2 },
                { "label": "F", "matrix": [2, 4], "x": 4.75, "y": 2 },
                { "label": "G", "matrix": [2, 5], "x": 5.75, "y": 2 },
                { "label": "H", "matrix": [2, 6], "x": 6.75, "y": 2 },
                { "label": "J", "matrix": [2, 7], "x": 7.75, "y": 2 },
                { "label": "K", "matrix": [2, 8], "x": 8.75, "y": 2 },
                { "label": "L", "matrix": [2, 9], "x": 9.75, "y": 2 },
                { "label": "Scolon", "matrix": [2, 10], "x": 10.75, "y": 2 },
                { "label": "Quote", "matrix": [2, 11], "x": 11.75, "y": 2 },
                { "label": "Enter", "matrix": [2, 13], "w": 2.25, "x": 12.75, "y": 2 },
                { "label": "PgDn", "matrix": [2, 14], "x": 15, "y": 2 },

                { "label": "Shift", "matrix": [3, 0], "w": 2.25, "x": 0, "y": 3 },
                { "label": "Z", "matrix": [3, 2], "x": 2.25, "y": 3 },
                { "label": "X", "matrix": [3, 3], "x": 3.25, "y": 3 },
                { "label": "C", "matrix": [3, 4], "x": 4.25, "y": 3 },
                { "label": "V", "matrix": [3, 5], "x": 5.25, "y": 3 },
                { "label": "B", "matrix": [3, 6], "x": 6.25, "y": 3 },
                { "label": "N", "matrix": [3, 7], "x": 7.25, "y": 3 },
                { "label": "M", "matrix": [3, 8], "x": 8.25, "y": 3 },
                { "label": "Comma", "matrix": [3, 9], "x": 9.25, "y": 3 },
                { "label": "Dot", "matrix": [3, 10], "x": 10.25, "y": 3 },
                { "label": "Back Slash", "matrix": [3, 11], "x": 11.25, "y": 3 },
                { "label": "Shift", "matrix": [3, 12], "w": 1.75, "x": 12.25, "y": 3 },
                { "label": "Up", "matrix": [3, 13], "x": 14, "y": 3 },
                { "label": "Fn", "matrix": [3, 14], "x": 15, "y": 3 },

                { "label": "Ctrl", "matrix": [4, 0], "w": 1.25, "x": 0, "y": 4 },
                { "label": "Gui", "matrix": [4, 1], "w": 1.25, "x": 1.25, "y": 4 },
                { "label": "Alt", "matrix": [4, 2], "w": 1.25, "x": 2.5, "y": 4 },
                { "label": "Space", "matrix": [4, 6], "w": 6.25, "x": 3.75, "y": 4 },
                { "label": "Gui", "matrix": [4, 10], "x": 11, "y": 4 },
                { "label": "Left", "matrix": [4, 12], "x": 13, "y": 4 },
                { "label": "Down", "matrix": [4, 13], "x": 14, "y": 4 },
                { "label": "Right", "matrix": [4, 14], "x": 15, "y": 4 }
            ]
        },
        "LAYOUT_65_ansi_blocker": {
            "layout": [
                { "label": "Esc", "matrix": [0, 0], "x": 0, "y": 0 },
                { "label": "1", "matrix": [0, 1], "x": 1, "y": 0 },
                { "label": "2", "matrix": [0, 2], "x": 2, "y": 0 },
                { "label": "3", "matrix": [0, 3], "x": 3, "y": 0 },
                { "label": "4", "matrix": [0, 4], "x": 4, "y": 0 },
                { "label": "5", "matrix": [0, 5], "x": 5, "y": 0 },
                { "label": "6", "matrix": [0, 6], "x": 6, "y": 0 },
                { "label": "7", "matrix": [0, 7], "x": 7, "y": 0 },
                { "label": "8", "matrix": [0, 8], "x": 8, "y": 0 },
                { "label": "9", "matrix": [0, 9], "x": 9, "y": 0 },
                { "label": "0", "matrix": [0, 10], "x": 10, "y": 0 },
                { "label": "Minus", "matrix": [0, 11], "x": 11, "y": 0 },
                { "label": "Equal", "matrix": [0, 12], "x": 12, "y": 0 },
                { "label": "Backspace", "matrix": [0, 13], "w": 2, "x": 13, "y": 0 },
                { "label": "PrtSc", "matrix": [0, 14], "x": 15, "y": 0 },

                { "label": "Tab", "matrix": [1, 0], "w": 1.5, "x": 0, "y": 1 },
                { "label": "Q", "matrix": [1, 1], "x": 1.5, "y": 1 },
                { "label": "W", "matrix": [1, 2], "x": 2.5, "y": 1 },
                { "label": "E", "matrix": [1, 3], "x": 3.5, "y": 1 },
                { "label": "R", "matrix": [1, 4], "x": 4.5, "y": 1 },
                { "label": "T", "matrix": [1, 5], "x": 5.5, "y": 1 },
                { "label": "Y", "matrix": [1, 6], "x": 6.5, "y": 1 },
                { "label": "U", "matrix": [1, 7], "x": 7.5, "y": 1 },
                { "label": "I", "matrix": [1, 8], "x": 8.5, "y": 1 },
                { "label": "O", "matrix": [1, 9], "x": 9.5, "y": 1 },
                { "label": "P", "matrix": [1, 10], "x": 10.5, "y": 1 },
                { "label": "Left Bracket", "matrix": [1, 11], "x": 11.5, "y": 1 },
                { "label": "Right Bracket", "matrix": [1, 12], "x": 12.5, "y": 1 },
                { "label": "Slash", "matrix": [1, 13], "w": 1.5, "x": 13.5, "y": 1 },
                { "label": "PgUp", "matrix": [1, 14], "x": 15, "y": 1 },

                { "label": "Caps Lock", "matrix": [2, 0], "w": 1.75, "x": 0, "y": 2 },
                { "label": "A", "matrix": [2, 1], "x": 1.75, "y": 2 },
                { "label": "S", "matrix": [2, 2], "x": 2.75, "y": 2 },
                { "label": "D", "matrix": [2, 3], "x": 3.75, "y": 2 },
                { "label": "F", "matrix": [2, 4], "x": 4.75, "y": 2 },
                { "label": "G", "matrix": [2, 5], "x": 5.75, "y": 2 },
                { "label": "H", "matrix": [2, 6], "x": 6.75, "y": 2 },
                { "label": "J", "matrix": [2, 7], "x": 7.75, "y": 2 },
                { "label": "K", "matrix": [2, 8], "x": 8.75, "y": 2 },
                { "label": "L", "matrix": [2, 9], "x": 9.75, "y": 2 },
                { "label": "Scolon", "matrix": [2, 10], "x": 10.75, "y": 2 },
                { "label": "Quote", "matrix": [2, 11], "x": 11.75, "y": 2 },
                { "label": "Enter", "matrix": [2, 13], "w": 2.25, "x": 12.75, "y": 2 },
                { "label": "PgDn", "matrix": [2, 14], "x": 15, "y": 2 },

                { "label": "Shift", "matrix": [3, 0], "w": 2.25, "x": 0, "y": 3 },
                { "label": "Z", "matrix": [3, 2], "x": 2.25, "y": 3 },
                { "label": "X", "matrix": [3, 3], "x": 3.25, "y": 3 },
                { "label": "C", "matrix": [3, 4], "x": 4.25, "y": 3 },
                { "label": "V", "matrix": [3, 5], "x": 5.25, "y": 3 },
                { "label": "B", "matrix": [3, 6], "x": 6.25, "y": 3 },
                { "label": "N", "matrix": [3, 7], "x": 7.25, "y": 3 },
                { "label": "M", "matrix": [3, 8], "x": 8.25, "y": 3 },
                { "label": "Comma", "matrix": [3, 9], "x": 9.25, "y": 3 },
                { "label": "Dot", "matrix": [3, 10], "x": 10.25, "y": 3 },
                { "label": "Back Slash", "matrix": [3, 11], "x": 11.25, "y": 3 },
                { "label": "Shift", "matrix": [3, 12], "w": 1.75, "x": 12.25, "y": 3 },
                { "label": "Up", "matrix": [3, 13], "x": 14, "y": 3 },
                { "label": "Fn", "matrix": [3, 14], "x": 15, "y": 3 },

                { "label": "Ctrl", "matrix": [4, 0], "w": 1.25, "x": 0, "y": 4 },
                { "label": "Gui", "matrix": [4, 1], "w": 1.25, "x": 1.25, "y": 4 },
                { "label": "Alt", "matrix": [4, 2], "w": 1.25, "x": 2.5, "y": 4 },
                { "label": "Space", "matrix": [4, 6], "w": 6.25, "x": 3.75, "y": 4 },
                { "label": "Alt", "matrix": [4, 9], "x": 10, "y": 4, "w":1.25 },
                { "label": "Gui", "matrix": [4, 10], "x":11.25, "y":4, "w":1.25 },
                { "label": "Left", "matrix": [4, 12], "x": 13, "y": 4 },
                { "label": "Down", "matrix": [4, 13], "x": 14, "y": 4 },
                { "label": "Right", "matrix": [4, 14], "x": 15, "y": 4 }
            ]
        },
        "LAYOUT_65_ansi_blocker_split_bs": {
            "layout": [
                { "label": "Esc", "matrix": [0, 0], "x": 0, "y": 0 },
                { "label": "1", "matrix": [0, 1], "x": 1, "y": 0 },
                { "label": "2", "matrix": [0, 2], "x": 2, "y": 0 },
                { "label": "3", "matrix": [0, 3], "x": 3, "y": 0 },
                { "label": "4", "matrix": [0, 4], "x": 4, "y": 0 },
                { "label": "5", "matrix": [0, 5], "x": 5, "y": 0 },
                { "label": "6", "matrix": [0, 6], "x": 6, "y": 0 },
                { "label": "7", "matrix": [0, 7], "x": 7, "y": 0 },
                { "label": "8", "matrix": [0, 8], "x": 8, "y": 0 },
                { "label": "9", "matrix": [0, 9], "x": 9, "y": 0 },
                { "label": "0", "matrix": [0, 10], "x": 10, "y": 0 },
                { "label": "Minus", "matrix": [0, 11], "x": 11, "y": 0 },
                { "label": "Equal", "matrix": [0, 12], "x": 12, "y": 0 },
                { "label": "Backspace", "matrix": [0, 13], "x": 13, "y": 0 },
                { "label": "Del", "matrix": [2, 12], "x": 14, "y": 0 },
                { "label": "PrtSc", "matrix": [0, 14], "x": 15, "y": 0 },

                { "label": "Tab", "matrix": [1, 0], "w": 1.5, "x": 0, "y": 1 },
                { "label": "Q", "matrix": [1, 1], "x": 1.5, "y": 1 },
                { "label": "W", "matrix": [1, 2], "x": 2.5, "y": 1 },
                { "label": "E", "matrix": [1, 3], "x": 3.5, "y": 1 },
                { "label": "R", "matrix": [1, 4], "x": 4.5, "y": 1 },
                { "label": "T", "matrix": [1, 5], "x": 5.5, "y": 1 },
                { "label": "Y", "matrix": [1, 6], "x": 6.5, "y": 1 },
                { "label": "U", "matrix": [1, 7], "x": 7.5, "y": 1 },
                { "label": "I", "matrix": [1, 8], "x": 8.5, "y": 1 },
                { "label": "O", "matrix": [1, 9], "x": 9.5, "y": 1 },
                { "label": "P", "matrix": [1, 10], "x": 10.5, "y": 1 },
                { "label": "Left Bracket", "matrix": [1, 11], "x": 11.5, "y": 1 },
                { "label": "Right Bracket", "matrix": [1, 12], "x": 12.5, "y": 1 },
                { "label": "Slash", "matrix": [1, 13], "w": 1.5, "x": 13.5, "y": 1 },
                { "label": "PgUp", "matrix": [1, 14], "x": 15, "y": 1 },

                { "label": "Caps Lock", "matrix": [2, 0], "w": 1.75, "x": 0, "y": 2 },
                { "label": "A", "matrix": [2, 1], "x": 1.75, "y": 2 },
                { "label": "S", "matrix": [2, 2], "x": 2.75, "y": 2 },
                { "label": "D", "matrix": [2, 3], "x": 3.75, "y": 2 },
                { "label": "F", "matrix": [2, 4], "x": 4.75, "y": 2 },
                { "label": "G", "matrix": [2, 5], "x": 5.75, "y": 2 },
                { "label": "H", "matrix": [2, 6], "x": 6.75, "y": 2 },
                { "label": "J", "matrix": [2, 7], "x": 7.75, "y": 2 },
                { "label": "K", "matrix": [2, 8], "x": 8.75, "y": 2 },
                { "label": "L", "matrix": [2, 9], "x": 9.75, "y": 2 },
                { "label": "Scolon", "matrix": [2, 10], "x": 10.75, "y": 2 },
                { "label": "Quote", "matrix": [2, 11], "x": 11.75, "y": 2 },
                { "label": "Enter", "matrix": [2, 13], "w": 2.25, "x": 12.75, "y": 2 },
                { "label": "PgDn", "matrix": [2, 14], "x": 15, "y": 2 },

                { "label": "Shift", "matrix": [3, 0], "w": 2.25, "x": 0, "y": 3 },
                { "label": "Z", "matrix": [3, 2], "x": 2.25, "y": 3 },
                { "label": "X", "matrix": [3, 3], "x": 3.25, "y": 3 },
                { "label": "C", "matrix": [3, 4], "x": 4.25, "y": 3 },
                { "label": "V", "matrix": [3, 5], "x": 5.25, "y": 3 },
                { "label": "B", "matrix": [3, 6], "x": 6.25, "y": 3 },
                { "label": "N", "matrix": [3, 7], "x": 7.25, "y": 3 },
                { "label": "M", "matrix": [3, 8], "x": 8.25, "y": 3 },
                { "label": "Comma", "matrix": [3, 9], "x": 9.25, "y": 3 },
                { "label": "Dot", "matrix": [3, 10], "x": 10.25, "y": 3 },
                { "label": "Back Slash", "matrix": [3, 11], "x": 11.25, "y": 3 },
                { "label": "Shift", "matrix": [3, 12], "w": 1.75, "x": 12.25, "y": 3 },
                { "label": "Up", "matrix": [3, 13], "x": 14, "y": 3 },
                { "label": "Fn", "matrix": [3, 14], "x": 15, "y": 3 },

                { "label": "Ctrl", "matrix": [4, 0], "w": 1.25, "x": 0, "y": 4 },
                { "label": "Gui", "matrix": [4, 1], "w": 1.25, "x": 1.25, "y": 4 },
                { "label": "Alt", "matrix": [4, 2], "w": 1.25, "x": 2.5, "y": 4 },
                { "label": "Space", "matrix": [4, 6], "w": 6.25, "x": 3.75, "y": 4 },
                { "label": "Alt", "matrix": [4, 9], "x": 10, "y": 4, "w":1.25 },
                { "label": "Gui", "matrix": [4, 10], "x":11.25, "y":4, "w":1.25 },
                { "label": "Left", "matrix": [4, 12], "x": 13, "y": 4 },
                { "label": "Down", "matrix": [4, 13], "x": 14, "y": 4 },
                { "label": "Right", "matrix": [4, 14], "x": 15, "y": 4 }
            ]
        },
```

Reply:
Hi @Deemen17 

After reviewing `matrix_diagram.md` for solder/rev1 I've amended add community layouts that this board can accomodate.
`LAYOUT_65_ansi_blocker` and `LAYOUT_65_ansi_blocker_split_bs` is equivalent to matrix_diagram's 'Blocker 6.25U', without and with split backspace respectively.

Would you be able to add, in `info.json`, layouts for the remaining key arrangements:
- 6.25U RWKL
- 6.25U RWKL split_bs
- 7U WKL
- 7U WKL split_bs
- 7U RWKL
- 7U RWKL split_bs
- Tsangan
- Tsangan split_bs
- Blocker 7U
- Blocker 7U split_bs

Reply:
Hi @dunk2k 

I added all the available layouts, please review any corrections if necessary! Thanks!